modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -5,7 +5,7 @@ endif::[]
 
 The Scheduler component enables you to trigger a flow when a time-based condition is met. You can configure it to be triggered at a regular interval or give it a more flexible cron expression. For example, a Scheduler might trigger an event to start a flow every 5 seconds.
 
-Schedulers follow the timezone used by the machine where the Mule runtime is running. If an application is running in CloudHub, the timezone that the Scheduler conforms to corresponds to the region in which the CloudHub worker is running.
+Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
```

Comment:
Mule > Mule runtime engine (Mule)
Since Mule concept is not introduced before in the first paragraph of the doc.